CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 3.8.0
+* NEW: Record fetch request breadcrumbs. See: https://github.com/getsentry/raven-js/pull/744
+* NEW: Record contentEditable input event breadcrumbs. See: https://github.com/getsentry/raven-js/pull/748
+* BUGFIX: Updated Typescript declaration file. See: https://github.com/getsentry/raven-js/pull/746
+* NEW: File size improvements for raven.min.js  See: https://github.com/getsentry/raven-js/pull/721
```

Comment:
3.7.0 to 3.8.0 is 21.2 KB->20.8 KB, the breadcrumb additions added a decent amount of code